packages/components/src/components/date-picker/date-picker.tsx

```diff
@@ -508,4 +477,86 @@ export class DatePicker {
       </Host>
     );
   }
+  private updateLocalizedDomBits = () => {
```

Comment:
The function name seems to be confusing and not representative of the work it does. 
change it to `updateDomOnLocalizationChange`